### ISSUE TITLE
ShineWiFi-X support

### ISSUE THREAD

**Issue by jarekgrzabel** [muppet3000/homeassistant-grott#26]
Hi

Does it work with ShineWiFi-X dongles? I cannot seem to find configuration instruction for that as if I go to the IP assigned to that dongle in my network by the web browser it won't connect. There doesn't seem to be any port open on it at all.

Thanks!

**Comment by muppet3000** [comment]
Yes it does - this is more a question for Grott than this integration, however I'm more than happy to answer it here.
I need to update my documentation with this information - so I'll leave this ticket open as a reminder to do that.

Instead of going to the dongle via it's local Web UI you can configure it to speak to Grott via the Growatt Server website.

If you take a look over at this comment: https://github.com/muppet3000/homeassistant-grott/issues/5#issuecomment-1545710398 and then the screen grab from myself 2 comments further down you should see how you can do this.

I hope that helps, let me know if you have any further issues.
